### PR TITLE
added the cluster id inside deployment controller

### PR DESCRIPTION
**Pull request by Mengying-Li**
Signed-off-by: Mengying <mengyinglimandy@gmail.com>

### CODE REVIEW

File: admiral/pkg/clusters/registry.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ func (r *RemoteRegistry) createCacheController(clientConfig *rest.Config, cluste
 	}
 
 	log.Infof("starting deployment controller clusterID: %v", clusterID)
-	rc.DeploymentController, err = admiral.NewDeploymentController(stop, &DeploymentHandler{RemoteRegistry: r}, clientConfig, resyncPeriod)
+	rc.DeploymentController, err = admiral.NewDeploymentController(stop, &DeploymentHandler{RemoteRegistry: r, ClusterID: clusterID}, clientConfig, resyncPeriod)
```
Comment (aattuluri) [inline admiral/pkg/clusters/registry.go:132]: Are there other controllers missing ClusterId?

Comment (aattuluri) [inline admiral/pkg/clusters/registry.go:132]: Are there other controllers missing ClusterId?

**Review by aattuluri — CHANGES_REQUESTED** [review]

File: admiral/pkg/clusters/registry.go (hunk 1 above)
Comment (Mengying-Li) [reply admiral/pkg/clusters/registry.go:132]: Would you like me to update all of them ? @aattuluri

Comment (aattuluri) [reply admiral/pkg/clusters/registry.go:132]: Yep, that one is fine.

**Pull request merged**